front/src/auth_components/urls.py

```diff
@@ -0,0 +1,27 @@
+"""
+URL configuration for front project.
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.2/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+from .views import signup, signin, reset_password_email
+
+app_name = "auth_components"
+
+urlpatterns = [
+    path('signup/', signup),
+    path('signin/', signin),
+    path('reset-password-email/', reset_password_email),
```

Comment:
Maybe rename `reset-password-email/` to `reset-password/`?

Reply:
I reserve the reset-password/ view for the form where the user will enter his new password once the 6-digit code has been emailed to him.